docs/source/user/api.rst

```diff
@@ -474,9 +474,11 @@ phone_number       string
 Batch user creation
 -------------------
 
+**Requires the user auth token (Bearer Token)**.
+
```

Comment:
I have added this like other API urls, but I think it would be nice to include an example working curl request; I personally find them useful and I assume the user would as well.
Let me know if I should create an issue for adding sample curl requests in the documentation. :smile: